slicer/SemiologyVisualization.py

```diff
@@ -156,19 +162,25 @@ def getInclusionsWidget(self):
     self.seizureSemiologyCheckBox.setEnabled(False)  # not implemented yet
 
     self.epilepsyTopologyCheckBox.setToolTip(
-      'When the paper selects a sample of patients based on their established'
-      ' epileptogenic zone and/or site of surgical resection (seizure onset zone),'
-      ' and describes the related seizure semiology, e.g. papers looking at TLE, FLE, OLE, etc.'
+      'When the paper pre-selects samples of patients based on their established epileptogenic zone'
+      '(site of surgical resection) or seizure onset zone (neurophysiological/anatomical), and'
+      'describes the related seizure semiology - e.g. articles looking at TLE, FLE, OLE.'
+      ''
+      'When unticked, ALL epilepsy topology data are excluded, EVEN if there are other approaches.'
```

Comment:
I think we can stick to lowercase.

Reply:
It's for emphasis, as above. Please leave for now